### DISCUSSION TITLE
Hi there! (Simple questions to know you)

### DISCUSSION THREAD

**Discussion by Marc-MMA** [flyntwp/flynt#457] [Q&A] [answered]
Hello,

Please forgive my English if it is not perfect (DeepL) and my noobism! I just discovered the discussion tab in GitHub^^

Your theme looks really but really cool and it fits perfectly with my working methods. Until then I was using Underscores, modify to my liking since 3 years and I use ACF pro too. I like to partition everything so what better than Flynt!

However I was a little reluctant to use it because I had the impression that the repo was not too active, but maybe I'm wrong?

I had a problem when installing NPM, I opened a ticket that remained unanswered, that's why I wondered if it was still active. I'll deal with it, maybe it's my version of Node that is too recent?

Have a good time! I'm looking forward to reading you :)

**Comment by timohubois** [comment, marked as answer]
Hi @Marc-MMA, in most cases people use the wrong Node version, when they trying to install Flynt (the current release needs Node 12 — see [Requirements](https://github.com/flyntwp/flynt#dependencies)). 

We are also working on a new Version of Flynt – stay tuned ✌️

**Reply by Marc-MMA** [reply]
All right! I am pleased with your answer.

Yes I understand about the Node version, I like to work with updated stuff because I'm not familiar with Docker and other to switch them easily.

Glad to know your theme is still supported! So I will build my next project with it! Hope to see you soon!
